paddle/operators/scatter_op.cu

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
the format of this copyright is different with others.

Reply:
done

Reply:
seems not changed

Reply:
changed back.